src/sqlite_back.cc

```diff
@@ -238,6 +238,20 @@ std::map<std::string, DbTypes> SqliteBack::ColumnTypes(std::string table) {
   return rtn;
 }
 
+std::set<std::string> SqliteBack::Tables() {
+  using std::set;
+  using std::string;
+  set<string> rtn;
+  std::string sql = "SELECT name FROM sqlite_master WHERE type='table';";
+  SqlStatement::Ptr stmt;
+  stmt = db_.Prepare(sql);
+  int i;
+  for (i = 0; stmt->Step(); ++i) {
```

Comment:
I did exactly what is used in the constructor: https://github.com/cyclus/cyclus/blob/develop/src/sqlite_back.cc#L45 Is there are reason to not follow this pattern?